index.njk

```diff
@@ -83,4 +82,6 @@ date: Last Modified
   }) }}
 </template>
 
+{% inline_js '_includes/recent_waiter.js' %}
```

Comment:
Note that other inlined JS is now also in the static directory, not the _includes dir.

Reply:
Then these are now exposed on the web site.  Why is that?  

Anyway, this here is inlined because it is running in a blocking manner to prevent render flicker on page load.  E.g. when you go back from a package page to the homepage.  (And you used this widget here before.)

Reply:
> Then these are now exposed on the web site.

Well there is no harm in that. Nothing in the static dir is used anywhere until it’s linked, and I figured inlining them is just another way of using a script. Perhaps the inline scripts should have their own subdir, that would make things more obvious. But just letting you know where all the other ones went so this PR can align with that.